blessed/sequences.py

```diff
@@ -40,7 +44,7 @@ def _sort_sequences(regex_seqlist):
     # types, it is feasible.
     # pylint: disable=bad-builtin
     #         Used builtin function 'filter'
```

Comment:
At this point, its time to agree with pylint and expand code using filter into list comprehension.